lib/utils.pm

```diff
@@ -803,6 +803,7 @@ sub power_action {
         $shutdown_timeout *= 3;
         record_soft_failure("boo#1057637 shutdown_timeout increased to $shutdown_timeout (s) expecting to complete.");
     }
+    $shutdown_timeout = get_var("DEBUG_SHUTDOWN") if (get_var('DEBUG_SHUTDOWN'));
```

Comment:
You can pass default value `get_var()`, so 
`$shutdown_timeout = get_var("DEBUG_SHUTDOWN", $shutdown_timeout)`
On top I would name variable differently, something like SHUTDOWN_TIMEOUT.
And if you don't use multiplier here, all 3 assignments above are useless, so we could adjust code, so adjustment is done only once.